src/hooks/quotes.js

```diff
@@ -15,14 +15,21 @@ export const useQuotes = () => {
     setNumQuotes(numQuotes);
   };
 
-  const searchQuotes = (searchTerm) => {
-    setQuotes(quotes);
-    setUrl(`https://ron-swanson-quotes.herokuapp.com/v2/quotes/${searchTerm}/1`);
-  };
-
   const handleClick = (url) => {
```

Comment:
You should refactor your handleClick or crate a new function that calls your getRonQuote and changes state. Then you can use the new function inside your useEffect as well. something like `  const fetchAQuote = () => {
    getRonQuote(numQuotes)
      .then(quotes => setQuotes(quotes));
  };`

`useEffect(() => {
    fetchAQuote();
  }, []);`

your getRonQuote doesn't need to take in a url. but it should take in number that can be changed by the user so they can get more than one quote